plugins/wallet_plugin/wallet_plugin.cpp

```diff
@@ -42,9 +42,10 @@ void wallet_plugin::plugin_initialize(const variables_map& options) {
       if (options.count("wallet-dir")) {
          auto dir = options.at("wallet-dir").as<boost::filesystem::path>();
          if (dir.is_relative())
-            wallet_manager_ptr->set_dir(app().data_dir() / dir);
-         else
-            wallet_manager_ptr->set_dir(dir);
+            dir = app().data_dir() / dir;
+         if( !bfs::exists(dir) )
```

Comment:
Do we have a style we prefer for if statements? In this one and many others we have a leading and trailing space. In others we don't.